lib/async_hooks.js

```diff
@@ -278,6 +288,11 @@ class AsyncResource {
     emitInitScript(
       this[async_id_symbol], type, this[trigger_async_id_symbol], this
     );
+
+    if (!opts.disableTracking) {
```

Comment:
Tracking/trace can mean so many things. Maybe call it `if (options.manualDestroy)`?

Reply:
Hmm, we don't actually mind manualDestroy since we prevent double destroys. What we actually dislike is a delayed (after GC) sensitive emitDestroy. Soo `delayedSensitiveDestroy`? May be too long...

Reply:
I don’t mind long names (I actually think they are a great idea and we need more of them), but `delayedSensitiveDestroy` doesn’t seem to say much imo … how about `requireManualDestroy` if you want to be explicit about what is optional/required?

Reply:
Sounds good to me.